### ISSUE TITLE
An error ocurred while starting the kernel

### ISSUE THREAD

**Issue by yangwencenc** [spyder-ide/spyder#22405]
### What steps will reproduce the problem?

The error is:

Traceback (most recent call last):
File "E:\Program\Anaconda3\lib\site‑packages\traitlets\traitlets.py", line 535, in get
value = obj._trait_values[self.name]
KeyError: 'context'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
File "E:\Program\Anaconda3\lib\site‑packages\spyder\plugins\ipythonconsole\plugin.py", line 1422, in create_kernel_manager_and_kernel_client
kernel_manager.start_kernel(stderr=stderr_handle,
File "E:\Program\Anaconda3\lib\site‑packages\jupyter_client\manager.py", line 342, in start_kernel
self.post_start_kernel(**kw)
File "E:\Program\Anaconda3\lib\site‑packages\qtconsole\manager.py", line 60, in post_start_kernel
super().post_start_kernel(**kw)
File "E:\Program\Anaconda3\lib\site‑packages\jupyter_client\manager.py", line 323, in post_start_kernel
self._connect_control_socket()
File "E:\Program\Anaconda3\lib\site‑packages\jupyter_client\manager.py", line 255, in _connect_control_socket
self._control_socket = self._create_connected_socket('control')
File "E:\Program\Anaconda3\lib\site‑packages\jupyter_client\connect.py", line 548, in _create_connected_socket
sock = self.context.socket(socket_type)
File "E:\Program\Anaconda3\lib\site‑packages\traitlets\traitlets.py", line 575, in __get__
return self.get(obj, cls)
File "E:\Program\Anaconda3\lib\site‑packages\traitlets\traitlets.py", line 538, in get
default = obj.trait_defaults(self.name)
File "E:\Program\Anaconda3\lib\site‑packages\traitlets\traitlets.py", line 1578, in trait_defaults
return self._get_trait_default_generator(names[0])(self)
File "E:\Program\Anaconda3\lib\site‑packages\jupyter_client\manager.py", line 64, in _context_default
return zmq.Context()
File "zmq/backend/cython/context.pyx", line 48, in zmq.backend.cython.context.Context.__cinit__
zmq.error.ZMQError: Unknown error

### Traceback
```python-traceback
  File "E:\Program\Anaconda3\lib\site-packages\spyder\plugins\editor\plugin.py", line 279, in report_open_file
    status = self.main.completions.start_client(language.lower())
  File "E:\Program\Anaconda3\lib\site-packages\spyder\plugins\completion\plugin.py", line 373, in start_client
    client_started = client_info['plugin'].start_client(language)
  File "E:\Program\Anaconda3\lib\site-packages\spyder\plugins\completion\languageserver\plugin.py", line 419, in start_client
    language_client['instance'] = LSPClient(
  File "E:\Program\Anaconda3\lib\site-packages\spyder\plugins\completion\languageserver\client.py", line 132, in __init__
    self.context = zmq.Context()
  File "zmq/backend/cython/context.pyx", line 48, in zmq.backend.cython.context.Context.__cinit__
zmq.error.ZMQError: Unknown error
Failed to send bug report on Github. response={'code': 401, 'json': {'message': 'Bad credentials', 'documentation_url': 'https://docs.github.com/rest', 'status': '401'}}
```

## Versions

* Spyder version: 4.2.5 
* Python version: 3.8.8
* Qt version: 5.12.1
* PyQt5 version: 5.12
* Operating System: Windows 7

### Dependencies

```

# Mandatory:
atomicwrites >=1.2.0            :  1.4.0 (OK)
chardet >=2.0.0                 :  4.0.0 (OK)
cloudpickle >=0.5.0             :  1.6.0 (OK)
diff_match_patch >=20181111     :  20200713 (OK)
intervaltree >=3.0.2            :  3.1.0 (OK)
IPython >=7.6.0                 :  7.22.0 (OK)
jedi =0.17.2                    :  0.17.2 (OK)
jsonschema >=3.2.0              :  3.2.0 (OK)
keyring >=17.0.0                :  22.3.0 (OK)
nbconvert >=4.0                 :  6.0.7 (OK)
numpydoc >=0.6.0                :  1.1.0 (OK)
paramiko >=2.4.0                :  2.7.2 (OK)
parso =0.7.0                    :  0.7.0 (OK)
pexpect >=4.4.0                 :  4.8.0 (OK)
pickleshare >=0.4               :  0.7.5 (OK)
psutil >=5.3                    :  5.8.0 (OK)
pygments >=2.0                  :  2.8.1 (OK)
pylint >=1.0                    :  2.7.4 (OK)
pyls >=0.36.2;<1.0.0            :  0.36.2 (OK)
pyls_black >=0.4.6              :  0.4.6 (OK)
pyls_spyder >=0.3.2             :  0.3.2 (OK)
qdarkstyle >=2.8;<3.0           :  2.8.1 (OK)
qtawesome >=0.5.7               :  1.0.2 (OK)
qtconsole >=5.0.3               :  5.0.3 (OK)
qtpy >=1.5.0                    :  1.9.0 (OK)
rtree >=0.8.3                   :  0.9.7 (OK)
setuptools >=39.0.0             :  52.0.0.post20210125 (OK)
sphinx >=0.6.6                  :  3.5.3 (OK)
spyder_kernels >=1.10.2;<1.11.0 :  1.10.2 (OK)
textdistance >=4.2.0            :  4.2.1 (OK)
three_merge >=0.1.1             :  0.1.1 (OK)
watchdog >=0.10.3;<2.0.0        :  1.0.2 (OK)
zmq >=17                        :  20.0.0 (OK)

# Optional:
cython >=0.21                   :  0.29.23 (OK)
matplotlib >=2.0.0              :  3.3.4 (OK)
numpy >=1.7                     :  1.20.1 (OK)
pandas >=1.1.1                  :  1.2.4 (OK)
scipy >=0.17.0                  :  1.6.2 (OK)
sympy >=0.7.3                   :  1.8 (OK)
```

**Comment by A-Infor** [comment]
Any reason to stick to Spyder 4.2.5 instead of using Spyder 6.0.0?

**Comment by ccordoba12** [comment]
Hey @yangwencenc, thanks for reporting. This problem should be fixed in our latest version available in Anaconda (5.5.1). Please update by closing Spyder, opening the Anaconda Prompt and running there the commands mentioned in [our documentation](http://docs.spyder-ide.org/current/installation.html#updating-with-anaconda).
